src/index.js

```diff
@@ -155,7 +157,7 @@ class LocalstackPlugin {
 
   readConfig() {
     this.config = (this.serverless.service.custom || {}).localstack || {};
-    Object.assign(this.config, this.options);
+    Object.assign({}, this.options, this.config);
```

Comment:
I'm assuming the purpose of this change is to give precedence to values in `this.config` over `this.options` (in case there are duplicate keys)..?

I think this should be:
```
this.config = Object.assign({}, this.options, this.config);
```
Otherwise we're not using the result (or side-effects) of the `Object.assign` call.